Assets/Scripts/ResourceSystem/ResourceHolder.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
```

Comment:
BuildingInfo can use prefab instead of a Sprite.

Reply:
Done